src/app/components/profile/profile.component.scss

```diff
@@ -1,96 +1,207 @@
 @import './variables.scss';
 @import './mixins.scss';
-
-.mid-container {
-	background-color: #fafafa;
-	.profile-container {
-		.card {
-			background: $gray-lighter;
-			padding-bottom:30px;
-			width:50%;
-			margin:0 auto;
-		}
-		font-size:$fs-16;
-		font-weight: $fw-light;
-		color:$gray-dark;
-		margin:30px;
-
-		.dark {
-			color:$gray-darker;
-		}
-		.highlight {
-			color: $yellow;
-		}
-		.item {
-			margin-bottom: 13px;
-			text-align: center;
-		}
-		.item button {
-			padding-top: 10px;
-			padding-bottom: 10px;
-		}
-		.profile-photo {
-			height:80px;
-			border-radius:50%;
-			@include box-shadow(0px, 0px, 5px, 0px, $overlay-light);
-		}
-		.user-details {
-			margin-top: 14px;
-			margin-bottom: 14px;
-		}
-		.change-password {
-			margin-top: 25px;
-			margin-bottom: 14px;
-			.change-password-text {
-				margin-right: 20px;
-				margin-bottom: 12px;
-			}
-		}
-		.modal-container {
-			&.hidden {
-				display:none;
-			}
-		}
-		.copy {
-			background: none !important;
-			border: none !important;
-			color: $red-light !important;
-			font-size: $fs-24 !important;
-			i {
-				margin-left:-20px;
-			}
-			&:hover {
-				box-shadow:none !important;
-				color: $red-medium;
-			}
-		}
-
+ 
+#card_container {
+	
+.card {
+	background: $gray-lighter;
+	padding-bottom:30px;
+	width:50%;
+	margin:0 auto;
+	}
+	font-size:$fs-16;
+	font-weight: $fw-light;
+	color:$gray-dark;
+	margin:30px;
+	.dark {
+		color:$gray-darker;
 	}
+	.highlight {
+		color: $yellow;
+	}
+	.item {
+		margin-bottom: 13px;
+		text-align: center;
+	}
+	.item button {
+		padding-top: 10px;
+		padding-bottom: 10px;
+	}
+	.profile-photo {
+		height:80px;
+	    border-radius:50%;
+		@include box-shadow(0px, 0px, 5px, 0px, $overlay-light);
+	}
+	.user-details {
+		margin-top: 14px;
+		margin-bottom: 14px;
+	}
+	.change-password {
+		margin-top: 25px;
+		margin-bottom: 14px;
+		.change-password-text {
+		margin-right: 20px;
+		margin-bottom: 12px;
+	}
+}
+.modal-container {
+	&.hidden {
+    display:none;
+}
+}
+.copy {
+	background: none !important;
```

Comment:
It will look more better if you try to give a blank line between two CSS :smiley:

Reply:
like after a css tag end?

Reply:
yes, like here just after the `.modal-container` css tag end.